general-superstaq/general_superstaq/finance.py

```diff
@@ -59,19 +63,33 @@ def read_json_maxsharpe(json_dict: gss.MaxSharpeJson) -> MaxSharpeOutput:
     return MaxSharpeOutput(best_portfolio, best_ret, best_std_dev, best_sharpe_ratio, qubo)
 
 
-class Finance:  # pylint: disable=missing-class-docstring
+class Finance:
+    """Overarching Finance class to implement methods above
+    __init__: Initializies superstaq client type.
+    submit_qubo: Submits the given QUBO to the target backend.
+    find_min_vol_portfolio: Finds the portfolio with minimum volatility that exceeds a specified
+    desired return.
+    find_max_pseudo_sharpe_ratio: Finds the optimal equal-weight portfolio from a possible pool of
+    stocks.
+    """
```

Comment:
nit: For this, a description of all the methods was not actually needed, but it can be helpful (though it's not necessary to reference the init constructor).

```suggestion
  """Overarching class for the finance endpoints.

    Methods
    -------
    submit_qubo: Submits the given QUBO to the target backend.
    find_min_vol_portfolio: Finds the portfolio with minimum volatility that exceeds a specified
    desired return.
    find_max_pseudo_sharpe_ratio: Finds the optimal equal-weight portfolio from a possible pool of
    stocks.
    """
```